metadata-ingestion/tests/integration/mlflow/test_mlflow_source.py

```diff
@@ -41,14 +43,20 @@ def pipeline_config(tracking_uri: str, sink_file_path: str) -> Dict[str, Any]:
 
 
 @pytest.fixture
-def generate_mlflow_data(tracking_uri: str) -> None:
+def generate_mlflow_data(tracking_uri: str, monkeypatch: MonkeyPatch) -> None:
+    test_uuid = "02660a3bee9941ed983667f678ce5611"
```

Comment:
https://github.com/datahub-project/datahub/pull/12587/files#diff-a8deda739eaafab77b16e103d159813fc353d9a915f3be39dd275ec139a18bedR552

I noticed that `id` and `outputUrls` here are dynamically generated for each --update-golden-files causing CI to fail

so i did a workaround with Monkeypatch -- making it to be static. 
seems like this works temporarily but not sure if this is the best approach. -- lmk your thoughs